cms/envs/common.py

```diff
@@ -117,6 +117,9 @@
     # for consistency in user-experience, keep the value of this feature flag
     # in sync with the one in lms/envs/common.py
     'IS_EDX_DOMAIN': False,
+
+    # Enable support for content libraries
```

Comment:
I would suggest adding a comment about enabling split as well. But I don't want to have to rerun this PR if there are no other issues....